src/patternfly/components/Form/styles.scss

```diff
@@ -0,0 +1,462 @@
+      > * {
+        flex: 0;
+        margin-right: var(--pf-c-form--MarginRight--inline);
+      }
+
+      // Label outside of pf-c-form__check
+      > .pf-c-form__label {
+        flex: 1 0 100%;
+        margin: 0;
+      }
+
+      .pf-c-form__label {
+        white-space: nowrap;
+      }
+    }
+  }
+
+  // Form label - pf-c-form__label
+  &__label {
+    display: flex;
+    align-items: baseline;
+    // using padding here to prevent flicker
+    padding-bottom: var(--pf-c-form__label--PaddingBottom);
+    margin: 0;
+    font-size: var(--pf-c-form__label--FontSize);
+
+    // Hide label selection on click
+    &::selection {
+      background: none;
+    }
+
+    // State - disabled
+    &.pf-m-disabled {
+      color: var(--pf-c-form__label--Color--disabled);
+    }
+
+    // Required
+    &.pf-m-required {
+      position: relative;
+
+      &::before {
+        position: absolute;
+        top: var(--pf-c-form__label--Top--required);
+        left: var(--pf-c-form__label--Left--required);
+        content: "\002A";
+      }
+    }
+  }
+
+  // Label group - pf-c-form__label-group
+  // Used when label contains elements like tooltips
+  &__label-group {
+    display: flex;
+
+    // Space 
+    > *:not(.pf-c-form__label) {
+      margin-left: var(--pf-c-form__label-group__item--MarginLeft);
+    }
+
+    &.pf-m-tooltip {
+      display: inline-flex;
+      align-items: first baseline;
+
+      > * {
+        margin-right: var(--pf-c-tooltip--MarginLeft);
+      }
+    }
+  }
+
+  // Text, textarea and select inputs
+  &__input[type="text"],
+  &__select select,
+  &__input[list],
+  &__textarea {
+    width: 100%;
+    padding: var(--pf-c-form__input--PaddingTop) var(--pf-c-form__input--PaddingRight) var(--pf-c-form__input--PaddingBottom) var(--pf-c-form__input--PaddingLeft);
+    font-size: var(--pf-c-form__input--FontSize);
+    line-height: var(--pf-c-form__input--LineHeight);
+    color: var(--pf-c-form__input--Color);
+    background: var(--pf-c-form__input--Background);
+    border: none;
+    box-shadow: inset 0 0 0 var(--pf-c-form__input--BorderSize) var(--pf-c-form__input--BorderColor);
+    transition: .2s;
+
+    // Hover
+    &:hover:not([disabled]) {
+      box-shadow: inset 0 0 0 var(--pf-c-form__input--BorderSize--hover) var(--pf-c-form__input--BorderColor);
+    }
+
+    // Focus
+    &:focus,
+    &:hover:focus {
+      outline: none;
+      box-shadow: inset 0 0 0 var(--pf-c-form__input--BorderSize--focus) var(--pf-c-form__input--BorderColor--focus);
+    }
+
+    // Disabled
+    &:disabled {
+      color: var(--pf-c-form__input--Color--disabled);
+      background-color: var(--pf-c-form__input--Background--disabled);
+      box-shadow: inset 0 0 0 var(--pf-c-form__input--BorderSize) var(--pf-c-form__input--BorderColor--disabled);
+
+      &:hover {
+        cursor: not-allowed;
+      }
+    }
+  }
+
+  // Text inputs - pf-c-form__input[type="text"]
+  &__input[type="text"] {
+    &.pf-m-error,
```

Comment:
Maybe we should talk with designers about this, but can't we make error handling more universal. So text areas, selects and etc can use the same styles?